pkg/objstore/swift/swift.go

```diff
@@ -247,9 +256,7 @@ func configFromEnv() SwiftConfig {
 func validateForTests(conf SwiftConfig) error {
 	if conf.AuthUrl == "" ||
 		conf.Username == "" ||
-		conf.Password == "" ||
-		(conf.ProjectName == "" && conf.ProjectID == "") ||
-		conf.RegionName == "" {
+		conf.Password == "" {
```

Comment:
@FlorinPeter One of `ProjectName`, `ProjectID` is mandatory for OpenStack client initialization. Any particular reason to relax these checks?

Reply:
@sudhi-vm I relaxed it because I had some issues with my credentials:
```
export OS_AUTH_URL=https://some.url.com/v3
export OS_USERNAME=username
export OS_PASSWORD=password
export OS_PROJECT_NAME=some-projekt-name
export OS_USER_DOMAIN_NAME=some-user-domain-name
```
But it looks like I removed maybe to much ;)